components/dropdown/dropdown-content-mixin.js

```diff
@@ -755,6 +789,27 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		await adjustPosition();
 	}
 
+	__removeRepositionHandlers() {
+		if (!this._fixedPositioning) return;
+		if (!this._scrollablesObserved) return;
+
+		this._scrollablesObserved.forEach(node => {
+			node.removeEventListener('scroll', this.__reposition);
+		});
+		this._scrollablesObserved = null;
+	}
+
+	__reposition() {
+		// throttle repositioning (https://developer.mozilla.org/en-US/docs/Web/API/Document/scroll_event#scroll_event_throttling)
+		if (!this.__repositioning) {
+			requestAnimationFrame(() => {
+				this.__position(true);
```

Comment:
This `true` tells it to ignore vertical when repositioning the content, so that it's not flipping as the user scrolls.

If we later decide we actually _do_ want it to flip, we can simply remove this `true` from here.